### ISSUE TITLE
[1.5.6] Pneumaticraft Refinery Output Crash

### ISSUE THREAD

**Issue by ahmetkca** [AllTheMods/ATM-6#1311]
I was playing with pneumaticcraft i had pressured chamber going and then I crafted refinery controller and refinery output
I placed refinery controller on the ground then i put the refinery output on top of refinery controller and when i break it my game just freeze and my game crashed

To reproduce 
Have pressurized chamber going insert 4 stacks of iron and start to produce air
then place a refinery controller somewhere close to pressurized chamber and then put the refinery output on top of the refinery controller and break the refinery ooutput

Logs
https://pastebin.com/GeRsLVQX

I tried to join to my world but then i got this crash
https://pastebin.com/xaATiAew

**Comment by Hotnastymilk** [comment]
Similar thing happened to me yesterday, and then was able to reproduce it today. I dont think the iron in the pressure chamber is causing the issue, it seems that having the Refinery Output block out in the world as a plop item will cause a crash. I imagine that just having it in your inventory and dropping it manually would cause the same crash but I have not tried that yet because I just want to avoid another crash mostly, but I will try it later for science reasons,
When I turned the world back on I expected to see the item on the ground or in my inventory since I had just mined it, but it was deleted somehow.
I think that for whatever reason the game doesnt allow for the item to exist unless it is either sitting in your inventory or placed like a block.
